scubagoggles/rego/Commoncontrols.rego

```diff
@@ -1055,39 +1312,108 @@ tests contains {
 ########################
 
 #
-# Baseline GWS.COMMONCONTROLS.9.1v0.3
+# Baseline GWS.COMMONCONTROLS.9.1
```

Comment:
GWS.COMMONCONTROLS.9.1 and 9.2, even with the policy API, are not able to be tested using ScubaGoggles. The baseline requires that "Highly privileged accounts" and "All sensitive user accounts" be enrolled. The best ScubaGoggles could currently do is tell you if users are allowed to enroll, it can't tell you who is actually enrolled. The reports API actually has this capability too, but we removed that from ScubaGoggles so users wouldn't get a false sense of security.

Reply:
done